src/main/webapp/resources/js/apis/analysis/analysis.js

```diff
@@ -3,15 +3,120 @@
  */
 import axios from "axios";
 
+const URL = `${window.TL.BASE_URL}analysis/ajax/`;
+
```

Comment:
This is kind of a general comment on JavaScript.  You will find it is better to pass an object as an argument than multiple parameters.  This is because if you need to add a new parameter you don't have to change all positions in your code that calls that function if they do not need that parameter.

Reply:
Fixed in 9c4e640d5c452bf1455f9b93a1b1499a418283e6